jetty-server/src/main/java/org/eclipse/jetty/server/handler/AllowSymLinkAliasChecker.java

```diff
@@ -99,4 +88,22 @@ public boolean check(String uri, Resource resource)
         return false;
     }
 
+    private Path unrollSymbolicLink(Path path) throws IOException
```

Comment:
Probably need to put some kind  visited path Set, so that we can detected loops?